src/controllers/expense.controller.js

```diff
@@ -0,0 +1,95 @@
+const expensesService = require('../services/expense.service');
+const expensesHelpers = require('../helpers/expense.helpers');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  const expenses = expensesService.getExpenses(userId, categories, from, to);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getExpenseById(id);
+
+  if (expensesHelpers.isExpenseExist(id, res)) {
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (
+    expensesHelpers.isUserExist(userId, res) ||
+    expensesHelpers.validateRequestBodyFields({
+      userId,
+      title,
+      amount,
+      category,
+      note,
+      res,
+    })
```

Comment:
```suggestion
    expensesHelpers.validateRequestBodyFields(req.body)
```
You can just pass body